.github/workflows/continuous-integration-lint.yml

```diff
@@ -5,33 +5,101 @@ on:
     types: [ opened, synchronize, reopened]
 
```

Comment:
We should add

```
  push:
    branches: [ main ]
```

too because SonarQube has to be run on the main branch for metrics